examples/js/loaders/FBXLoader2.js

```diff
@@ -571,11 +571,10 @@
 
 							var faceVertexBuffer = [];
 							var polygonIndex = 0;
-							for ( var polygonVertexIndex = 0; polygonVertexIndex < indexBuffer.length; ++ polygonVertexIndex ) {
+							indexBuffer.forEach( function ( vertexIndex, polygonVertexIndex, indexBuffer ) {
```

Comment:
In the Mr.doob's Code Style...

> Don't use foreach loops.

https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2#cycles